src/utils/constants/abortName.ts

```diff
@@ -0,0 +1 @@
+export const abortName = 'AbortError'
```

Comment:
global constant name as a rule should be upper case `ABORT_ERROR = 'AbortError'`

please, pay attention. Why `AbortError` variables named `abortName`?